sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/View.java

```diff
@@ -263,8 +263,8 @@ private AsList() {}
 
       // TODO(BEAM-10097): Make this the default expansion for all portable runners.
       if (hasExperiment(input.getPipeline().getOptions(), "beam_fn_api")
-          && (hasExperiment(input.getPipeline().getOptions(), "use_runner_v2")
-              || hasExperiment(input.getPipeline().getOptions(), "use_unified_worker"))) {
+          || hasExperiment(input.getPipeline().getOptions(), "use_runner_v2")
```

Comment:
Is it possible for us to only check `beam_fn_api` here and below since we now are thinking `beam_fn_api` == `use_runner_v2` == `use_unified_worker`.

Reply:
That's a good idea. Done here.

I plan to follow up with removing the Fn API overrides in the DataflowRunner, since the v1beta3 graph is no longer used with Fn API. Then we may also be able to eliminate the flags there. I did not add it to this PR since it can be done separately, to prevent scope increase for this PR.

Reply:
Sounds good. Thanks for the explanation.